src/js/typography.js

```diff
@@ -0,0 +1,119 @@
+const FontFaceObserver = require('fontfaceobserver/fontfaceobserver.standalone.js');
+const superstore = require('superstore-sync');
+
+class Typography {
+
+	constructor (typographyEl, opts) {
+		this.typographyEl = typographyEl;
+
+		this.opts = opts || Typography.getOptions(typographyEl);
+		this.opts = Typography.checkOptions(this.opts);
+
+		this.fontConfigs = [
+			{
+				family: 'FinancierDisplayWeb',
+				weight: 'normal',
+				label: 'serifDisplay'
+			},
+			{
+				family: 'MetricWeb',
+				weight: 'normal',
+				label: 'sans'
+			},
+			{
+				family: 'MetricWeb',
+				weight: 600,
+				label: 'sansBold'
+			},
+			{
+				family: 'FinancierDisplayWeb',
+				weight: 700,
+				label: 'serifDisplayBold'
+			}
+		];
+
+		this.loadFonts();
+	}
+
+	/**
+	 * Get the data attributes from the typographyEl. If typography is being set up
+	 * declaratively, this method is used to extract the data attributes from
+	 * the DOM.
+	 * @param {HTMLElement} typographyEl - The typography element in the DOM (Required)
+	*/
+	static getOptions(typographyEl) {
+		const dataset = typographyEl.dataset;
```

Comment:
Quite possibly, will see if I changed anything from `o-tooltip` or if it stayed the same